packages/default/scss/chat/_layout.scss

```diff
@@ -414,6 +414,61 @@ $message-group-spacing: calc( #{$chat-avatar-size} + #{$chat-avatar-spacing} );
 
 
 
+@include exports( "chat/toolbar/layout" ) {
+
+    .k-chat .k-toolbar-box {
+        padding: $chat-toolbar-padding-y $chat-toolbar-padding-x;
+        width: 100%;
+        box-sizing: border-box;
+        overflow: hidden;
+        position: relative;
+        flex: 0 0 auto;
+
+        .k-button-list {
+            // margin-bottom: -20px;
+            // padding-bottom: 20px;
```

Comment:
Remove the commented out code, or add a comment why these lines are commented out.

Reply:
will do